bin/strip-private-prefix

```diff
@@ -1,23 +1,48 @@
 #!/bin/sh
 set -e
 
-if [ $# -eq 0 ]; then
-    echo "No arguments provided"
-    exit 1
-fi
+unset IS_CONVO
+unset PREFIX
+
+for arg in "$@"; do
+  case $arg in
+    --conversations)
+      export IS_CONVO=true
+      ;;
+    -*)
+      echo "Invalid flag $arg was provided"
+      exit 1
+      ;;
+    *)
+      if [ -z ${PREFIX+x} ]; then
+        export PREFIX=$(echo $arg | perl -p -e "s/@/\\\@/g")
+      else
+        echo "Invalid argument $arg was provided"
+        exit 1
+      fi
+      ;;
+  esac
+done
 
-export PREFIX=$(echo $1 | perl -p -e "s/@/\\\@/g")
+if [ -z ${PREFIX+x} ]; then
+  echo "No prefix provided"
+  exit 1
+fi
 
-echo -n "Stripping private repo prefix '$PREFIX' from package.json and source files .. "
+echo "Stripping private repo prefix from package.json and source files: $PREFIX"
```

Comment:
I wouldn't change that line.

1. prefix is inside '' so if we accidentally lost some checks and  it happens to be empty we will see it. If empty  prefix is at the end of the line we will simply miss it. `Stripping private repo prefix from package.json and source files:` looks like just a regular line.
2. two dots at the end showing the operation is proceeding.